File: CRM/src/app/shared/eHospital_DongNai_A/DM_NgayNghiLe.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DM_NgayNghiLe } from './DM_NgayNghiLe.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DM_NgayNghiLeService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'NgayLe_Id', 'Nam', 'NgayNghi', 'GhiChu', 'NguoiTao_Id', 'NgayTao', 'NguoiCapNhat_Id', 'NgayCapNhat'];

    List: DM_NgayNghiLe[] | undefined;
    ListFilter: DM_NgayNghiLe[] | undefined;
    FormData!: DM_NgayNghiLe;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DM_NgayNghiLe";
    }
}

File: Service_eHospital_DongNai_A/Implement/DM_NgayNghiLeService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class DM_NgayNghiLeService : BaseService<DM_NgayNghiLe, IDM_NgayNghiLeRepository>
    , IDM_NgayNghiLeService
    {
    private readonly IDM_NgayNghiLeRepository _DM_NgayNghiLeRepository;
    public DM_NgayNghiLeService(IDM_NgayNghiLeRepository DM_NgayNghiLeRepository) : base(DM_NgayNghiLeRepository)
    {
    _DM_NgayNghiLeRepository = DM_NgayNghiLeRepository;
    }
    }
    }

